azure-iot-hub-devicesdk/tests/aio/test_async_clients.py

```diff
@@ -18,38 +18,32 @@
 pytestmark = pytest.mark.asyncio
 
 
-async def completed_future():
-    f = asyncio.Future()
-    f.set_result = None
-    return f
-
-
-class FakeAsyncTransport(AbstractTransport):
+class FakeTransport(AbstractTransport):
```

Comment:
>FakeTransport [](start = 6, length = 13)

I assume the sync tests have a similar object.  Is it worth factoring this out into it's own .py file?

Reply:
Now that there is no specific Async Transport, they should be factored out into a shared pytest utility, yeah.

---
In reply to: [262237668](https://github.com/Azure/azure-iot-sdk-python-preview/pull/55#discussion_r262237668) [](ancestors = 262237668)